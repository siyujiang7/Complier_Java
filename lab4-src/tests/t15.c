long f(long g) {

}

long main() {
  f();
}
