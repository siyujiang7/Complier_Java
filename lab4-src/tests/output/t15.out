----------
 Errors/Warnings: 1 
----------
0 : Code can't precede declarations
0 : Break not in loop
0 : Continue not in loop
0 : Returning value in void function
0 : Incompatible return type
0 : Incompatible type assigning
0 : Assigning to an undeclared variable
0 : Non-Integer array subscript
0 : Assigning pointer to integer
0 : Undeclared Variable in expression
0 : Divide by zero
0 : Invalid unary operation
0 : Invalid operands to binary operator
0 : Variable/Function Redeclaration
0 : Incompatible function argument types
1 : Too few function arguments
0 : Too many function arguments
